frontend/src/components/Button.tsx

```diff
@@ -54,7 +54,7 @@ const Button = ({
   return (
     <>
       <Tooltip title={disabled ? 'Please add missing options' : ''}>
-        <span>
+        <span className={className}>
```

Comment:
@Mister-Corn is this going to break anything?  Without this, the `submit` button on the `postreg` page is at the top of the screen.